src/main/resources/i18n/messages.properties

```diff
@@ -995,8 +995,11 @@ ProjectMetadataTemplates.fields={0} fields
 
 MetadataFields.title=Metadata Fields
 
-MetadataFieldsList.create=Create Template
-MetadataFieldsList.empty=No metadata has been added to this project.
+MetadataFieldsListManager.create=Create Template
+MetadataFieldsListMember.empty=No metadata has been added to this project.
+MetadataFieldsListManager.restrictions=Restrictions
+MetadataFieldsListManager.restrictions-help=Each Metadata Field can be set to only be viewed by certain project roles.  Manager's will be able to view all fields while the Collaborators will only be able to view the fields that are selected for that role.
+server.MetadataFieldsListManager.update=Role for {0} update to {1}
```

Comment:
Changed `update` to `updated` in the actual text for the translation key

Reply:
Fixed in 0e81323